src/js/plugins/plugin.autoplay.js

```diff
@@ -0,0 +1,127 @@
+/*global BookReader */
+
+/**
+ * Plugin which adds an autoplay feature. Useful for kiosk situations.
+ */
+jQuery.extend(BookReader.defaultOptions, {
+  enableAutoPlayPlugin: true
+});
+
+/**
+ * @override BookReader.setup
+ */
+BookReader.prototype.setup = (function(super_) {
+  return function (options) {
+    super_.call(this, options);
+
+    this.auto      = false;
+    this.autoTimer = null;
+    this.flipDelay = 5000;
+  };
+})(BookReader.prototype.setup);
+
+/**
+ * @override BookReader.init
+ */
+BookReader.prototype.init = (function(super_) {
+  return function (options) {
+    super_.call(this, options);
+
+    if (!this.options.enableAutoPlayPlugin) return;
+    this.bind(BookReader.eventNames.stop, () => this.autoStop());
+  };
+})(BookReader.prototype.init);
+
+/**
+ * @override BookReader.bindNavigationHandlers
+ */
+BookReader.prototype.bindNavigationHandlers = (function(super_) {
+  return function() {
+    super_.call(this);
+
+    if (!this.options.enableAutoPlayPlugin) return;
+
+    // Note the mobile plugin attaches itself to body, so we need to select outside
+    const jIcons = this.$('.BRicon').add('.BRmobileMenu .BRicon');
+
+    jIcons.filter('.play').click(() => {
+      this.autoToggle();
+      return false;
+    });
+
+    jIcons.filter('.pause').click(() => {
+      this.autoToggle();
+      return false;
+    });
+  };
+})(BookReader.prototype.bindNavigationHandlers);
+
+/**
+ * Starts autoplay mode
+ * @param {object} overrides - accepts numeric properties flipSpeed and flipDelay
```

Comment:
Jsdoc also supports:

```suggestion
 * @param {object} overrides
 * @param {number} overrides.flipSpeed
 * @param {number} overrides.flipDelay
```